lib/streamlit/elements/widgets/number_input.py

```diff
@@ -444,7 +444,7 @@ def _number_input(
                 float_value = True
 
         if format is None:
-            format = "%d" if int_value else "%0.2f"
+            format = "%d" if int_value else "%0.2f"  # noqa: A001
```

Comment:
what's the complaint here and why can't it be fixed?

Reply:
I updated it to not require the noqa